node_helper.js

```diff
@@ -559,6 +570,25 @@ module.exports = NodeHelper.create({
     return [];
   },
 
+  /* NASA APIs documented under https://api.nasa.gov/.
+    This accesses the NASA Image and Video Library. 
+    Currently, it only loads the thumbnails, which in most cases have good enough quality.
+    For NASA API usage without an API key, there are hourly limits of about 1,000 requests. 
+    */
+  processNasaData: function (config, data) {
+    let images = [];
+    for (const image of data.collection.items) {
+      if (image?.links && image?.links?.length > 0) {
```

Comment:
Can this api return videos as well?  I see `"media_type": "image"` in the response data, should probably filter the items by that.